src/gwc/src/main/java/org/geoserver/gwc/controller/GwcWmtsRestUrlHandlerMapping.java

```diff
@@ -85,6 +90,13 @@ private static final class Wrapper extends HttpServletRequestWrapper {
 
         Wrapper(HttpServletRequest request, Catalog catalog, String workspaceName) {
             super(request);
+
+            // spring uses this LOOKUP_PATH
```

Comment:
```suggestion
            // Adjust LOOKUP_PATH used by spring to remove workspace
```

I appreciate small comments when the code is not clear

Reply:
changed